### ISSUE TITLE
New Install on Ubuntu server 18 lts

### ISSUE THREAD

**Issue by Fuquan999** [modoboa/modoboa#1727]
# Impacted versions

* Modoboa: X.X.x
* installer used: Yes/No ?
* Webserver: Nginx/Apache

# Steps to reproduce

# Current behavior

# Expected behavior

# Video/Screenshot link (optional)

**Comment by cremesk** [comment]
what is the issue?

**Comment by Fuquan999** [comment]
Hi All

Hoping someone can fill me in on the blanks on this one. I have just installed Modoboa email and at the end of the install I get the "Congratulations ....." but when I try to access the server using the https://mail. part I am not able to go anywhere. Take off the "mail." part from the URL and the servers Nginx web page is reachable.

If someone could point me in the right direction as to how to sort this issue out that would be appreciated as I cannot access the email server. The box is a public facing "standalone" server and it has a valid/legal static IP address to it as well as domain name so that should not be the problem as once you remove the word "mail." from the URL the server is reachable using https but not the email part. If anyone knows what may be wrong here that assistance would be appreciated.

Thanks]
Fuquan

**Comment by jammo-nl** [comment]
From your description is looks like you do not have a DNS entry for your mail host..
Does not look like a modoboa bug, but a configuration issue..

**Comment by Fuquan999** [comment]
Hi Jammo,

Thanks for the feedback. I did check in the dns records and although I had an entry both A and MX records there was a second entry there for some smtp as both MX and A records which after I deleted them then Modoboa was accessible on the admin side.

However, if any one can still help me identify where to locate the "settings.py" file in order to make custom changes as I cant seem to find the file. (as there are numerous "settings.py" files) The documentation on this is a bit faint and hard to follow directory paths as the docs does not really stipulate the actual paths (I am reading the 1.5.3 (online documentation on how to customize things).

If anyone knows where the actual path to the correct file for customizing the logo etc may be could some one point me there. Also where may I ask are the main Modoboa files stored is it in the "srv" folder or where?

Thanks
Fuquan
